packages/beacon-node/src/node/nodejs.ts

```diff
@@ -66,6 +66,7 @@ enum LoggerModule {
   backfill = "backfill",
   chain = "chain",
   eth1 = "eth1",
+  executionEngine = "execution_engine",
```

Comment:
what about just using `execution`? I feel like this is too long and inconsistent with other module names.

For CLI options we also just use `execution.` prefix

https://github.com/ChainSafe/lodestar/blob/96820d72b14fdeb4b1c21f6f1c6c8dc604bc8331/packages/cli/src/options/beaconNodeOptions/execution.ts#L6